### ISSUE TITLE
[Feature Request] The Ability to Search Canned Responses

### ISSUE THREAD

**Issue by BitcoinKid9** [osTicket/osTicket#5808]
Hello osTicket Support,

I'm running osTicket Version v1.15.1

My help desk has 4 pages of canned responses and would really appreciate to be able to search my canned responses within seconds rather than having to go through hundreds of canned responses one at a time until I find the right one in order to update the canned response with changes needed.

What can take 30 minutes can be reduced to just a couple seconds with a simple search in canned responses.

Please add this to the Feature Request for next update.

Thank you!

Regards,
BlackHawk17
RemoteTech

**Comment by mlipok** [comment]
do you mean to search o n this page:
https://...........pl/scp/canned.php

or at the time when you are answering in tickets.

**Comment by BitcoinKid9** [comment]
Hi, when I'm answering the tickets I'll be using multiple canned replies at times and usually after I close a ticket I will go back and change/update some of the canned replies on the 

https://........pl/scp/canned.php   (page)

but I will have to hunt and search through sometimes pages for the right canned reply where if I could just search for a keyword the system should find it in less than a second. I'll be honest most of the time I just go to cPanel and then phpmyadmin and open the database and do a search on the canned reply table and modify it right in the database to save time. I don't do this all day long but it can happen a few times a day as we keep fine tuning our replies for our valued customers. Thank you for looking into this.

Regards,
RemoteTech

**Comment by mlipok** [comment]
oh I see
in FAQ there is search feature
![image](https://user-images.githubusercontent.com/11089482/108363880-b571ea80-71f5-11eb-99e9-9613c8441cc9.png)

but in Canned Responses there is lack of search feature:
![image](https://user-images.githubusercontent.com/11089482/108363958-cae71480-71f5-11eb-9a64-f22b15300517.png)

And I undrestand and confirm that this will be difficult to search/edit/manage all the canned responses in case when you have 4 page of canned response to search them organoleptically (manually+visually) ... without the search feature.

**Comment by BitcoinKid9** [comment]
Thank you kind sir. :)

**Comment by BitcoinKid9** [comment]
Hello, has this feature request been implemented into the new latest release osTicket Version v1.15.2 available 7 days ago?
Thank you!

Regards,
RemoteTech

**Comment by JediKev** [comment]
@remotetech 

No, there would have been a pull request linked here as well as this issue being closed. If there is no update on this thread assume nothing has been done yet.

Cheers.
